docs/developer-guide/maps-configuration.md

```diff
@@ -182,6 +183,220 @@ in `localConfig.json`
 }
 ```
 
+#### WMTS
+WMTS Layer require a source object in the `sources` object of the map configuration where to retrieve the `tileMatrixSet`. The source is identified by the `capabilitiesURL`.
```

Comment:
Do we have a fallback for old maps? If capabilitiesURL is not present, will it work as before?

Reply:
Yes, we have. I specified it in the KVP example. Maybe is a good thing if I specify it also here.

Reply:
See LayerUtils.js => `getSourceId`